types/culori/src/a98/definition.d.ts

```diff
@@ -20,6 +20,6 @@ interface A98DefinitionMixin {
     };
 }
 
-declare const definition: Omit<typeof rgb, keyof A98DefinitionMixin> & A98DefinitionMixin;
+declare const modeA98: Omit<typeof rgb, keyof A98DefinitionMixin> & A98DefinitionMixin;
```

Comment:
Here and elsewhere, it made CJS bundling a little easier to just use the public names, even if they differ from source. We also do this in a few other places, such as ignoring internal functions from types altogether